### PR TITLE
Show hint, if error is about async on older Node.JS

### PR DESCRIPTION
**Pull request by floatdrop**
Not sure about `/\s+async\s+/.test(err.stack)` part, but it should be sufficient.

### CODE REVIEW

File: lib/handler.js [hunk 1]
```diff
@@ -25,15 +12,17 @@ module.exports = file => {
       mod = mod.default
     }
   } catch (err) {
-    if (!checkAsyncAwait()) {
+    logError(`Error when importing ${file}: ${err.stack}`, 'invalid-entry')
```
Comment (leo) [inline lib/handler.js:15]: Newline after this please

File: lib/handler.js [hunk 2]
```diff
@@ -25,15 +12,17 @@ module.exports = file => {
...
+    if (
+      err instanceof SyntaxError &&
+      /\s+async\s+/.test(err.stack) &&
+      Number(process.versions.node.split('.')[0]) < 8
+    ) {
       logError(
-        'In order for `async` & `await` to work, you need to use at least Node.js 8!',
+        'Hint: In order for `async` & `await` to work, you need to use at least Node.js 8!',
```
Comment (leo) [inline lib/handler.js]: Remove the "Hint:", please. The utility is printing its own prefix.

**Review by leo — CHANGES_REQUESTED** [review]

File: lib/handler.js (hunk 1 above)
Comment (floatdrop) [reply lib/handler.js:15]: Done.

File: lib/handler.js (hunk 2 above)
Comment (floatdrop) [reply lib/handler.js]: Done.

**Comment by leo** [comment]
Thank you!

**Pull request merged**